instagram-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import withAuthenticate from './authentication/withAuthenticate';
```

Comment:
I know you're almost there for the login page. If you're still having trouble tomorrow, try asking Kedasha for help. We managed to fix her page today so she should be able to assist.